packages/conventional-changelog-angular/conventional-recommended-bump.js

```diff
@@ -20,7 +20,9 @@ module.exports = {
 
     return {
       level: level,
-      reason: `There are ${breakings} BREAKING CHANGES and ${features} features`
+      reason: breakings === 1 ?
+        `There are ${breakings} BREAKING CHANGE and ${features} features` :
```

Comment:
`There are` => `There is`

Reply:
But There are Change and features....

Reply:
Ah, okay. I can see that. Please ignore my earlier comment.

Reply:
We can probably use https://github.com/sindresorhus/plur for pluralisation?

Reply:
Yeah sure, but it would be quite some work to extract all strings to be pluralized ^^ because I wouldn't just use it for this case @stevemao

Reply:
I definitely think we should incorporate `plur` into `conventional-changelog` to improve messaging, but as noted by @flxwu, it's probably a significant refactor that needs to be done separately from this particular scoped change.